lib/src/models/config_models/feature_active_config.dart

```diff
@@ -13,6 +13,7 @@ class FeatureActiveConfig {
     this.lastSeenAgoBuilderVisibility = true,
     this.receiptsBuilderVisibility = true,
     this.enableOtherUserName = true,
+    this.enableScrollToBottomButton = true,
```

Comment:
The default value will be false here.

Reply:
I was thinking of making it true by default, but now I think we will make it false for now. In the next major version, we can mark it as true by default.